lib/frizz/local.rb

```diff
@@ -10,7 +10,8 @@ def files
       @files ||= begin
         Dir.chdir(root_path) do
           Dir["**/*"].map do |local_path|
-            File.new(expand_path(local_path), local_path) unless ignore?(local_path)
+            File.new(expand_path(local_path), local_path) unless ignore?(local_path) ||
+                                                                 ignore_for_gzip_version?(local_path)
```

Comment:
Why not just to modify `ignore?` to also account for the gzip version? Since it's a private method only used here, nothing else depends on its behavior so you're free to modify it.. Also I think it makes sense as correct behavior of `ignore?` to tell us whether or not to ignore the file regardless of gzip, which I'd consider an internal detail of `ignore?`